joomla/joomla-cms
# [4.0] Move executable code out of index.php files

Pull Request for Issue #14174

### Summary of Changes

Similar to how we've already restructured the install app, the `administrator/index.php` and `index.php` files should be restructured in a way that allows those files to remain PHP 5.2 friendly while not impeding us using PHP 5.3+ features (like namespaced classes).

### Testing Instructions

- Install 4.0 using this branch (https://github.com/zero-24/joomla-cms/archive/indexphp-40.zip)
- confirm the frontend and backend works.

### Documentation Changes Required

None
